lib/proxy/mutators/urlencoded_bodyparser.js

```diff
@@ -0,0 +1,19 @@
+var body_parser = require('body-parser').urlencoded({ extended: true, limit: '1mb' });
+
+var module_tag = {
+  module: require('../../logger.js').getModulePath(__filename)
+};
+
+/**
+ * Parse the body of the request for urlencoded form-data.  Note that this will
+ * fail if the body is not utf-8.
+ */
+module.exports = function(proxy) {
+  return function(req, res, next) {
+    body_parser(req, res, function(err) {
+      if (err && err.status === 413) return next(err);
```

Comment:
Creates issue: `body_parser.urlencoded()` middleware throws an error response for content that is not charset === `utf-8` (with the dash), ever since our upgrade (we went body-parser 1.0.2->1.14.0) in PR #12.